src/visualization/types/Graph/view.tsx

```diff
@@ -64,6 +66,9 @@ const XYPlot: FC<Props> = ({properties, result, timeRange, annotations}) => {
   )
 
   const dispatch = useDispatch()
+  const annotationsModeIsActive = useSelector(
+    (state: AppState) => state.userSettings.showAnnotationsControls
```

Comment:
know of any more of these one off user settings? i've been collecting them: https://github.com/influxdata/ui/blob/master/src/shared/components/TimeZoneDropdown.tsx#L12